dashboard/views.py

```diff
@@ -150,3 +152,37 @@ def post(self, request):
         return Response(
             data=enrollment.json
         )
+
+
+class UnEnrollPrograms(APIView):
+    """
+    api that unenroll user from one ore more programs
+    """
+    authentication_classes = (
+        authentication.SessionAuthentication,
+        authentication.TokenAuthentication,
+    )
+    permission_classes = (permissions.IsAuthenticated, )
+
+    def post(self, request):
+        """
+        unenroll from MM program(s)
+        """
+        program_ids_serializer = UnEnrollProgramsSerializer(data=request.data)
+        program_ids_serializer.is_valid(raise_exception=True)
+        program_ids = program_ids_serializer.data['program_ids']
```

Comment:
This code should all get moved into the serializer, otherwise the serializer is just being used to parse a list of ints.